samples/Network/no-public-ip-except-for-one-subnet/README.md

```diff
@@ -1,7 +1,7 @@
 # Only Allow public IP in specific subnets
 
 Only allow public IP to be used in specific subnets
-
+Policy needs to be provided with a list of subnets in format of resource ID (e.g /subscriptions/{subscription_id}/resourcegroups/{resource_group)/providers/microsoft.network/virtualnetworks/{vnet}/subnets/{subnet-name}
```

Comment:
Please add the same information to the "description" field of the subnetIds parameter in both azurepolicy.json and azurepolicy.parameters.json